### PR TITLE
i18n SC-4702 Translations javascripts

### PR DESCRIPTION
**Pull request by Tratori**
Base links to copy

- https://ticketsystem.schul-cloud.org/browse/SC-3560 (main ticket)
- https://ticketsystem.schul-cloud.org/browse/SC-4702 (ticket regarding this PR)

### CODE REVIEW

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5480799&bid=17961678)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5480799&bid=17961678)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5480799&bid=17961678)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 10
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5480799&bid=17961678)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 24
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5480799&bid=17961678)

File: controllers/content.js [hunk 1]
```diff
@@ -9,35 +9,35 @@ const api = require('../api');
...
 router.use(authHelper.authChecker);
 
-const contentFilterSettings = () => [
+const contentFilterSettings = (res) => [
 	{
 		type: 'sort',
-		title: 'Sortierung',
-		displayTemplate: 'Sortieren nach: %1',
+		title: res.$t('content.headline.sortBy'),
+		displayTemplate: res.$t('content.label.sortBy'),
```
Comment (BlackGhostHD) [inline controllers/content.js:16]: Does it work with %1? Have you tried that?

Comment (Tratori) [reply controllers/content.js:16]: sortby%1
works :)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5480799&bid=17961678)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5480799&bid=17961678)

**Pull request merged**